### ISSUE TITLE
Different settings for quotes inside selector vs values ?

### ISSUE THREAD

**Issue by thierrymichel** [sasstools/sass-lint#1017]
Hi,

First of all, thanks for the great work!

Is there a way to have single quotes for declarations/properties

```yaml
attribute-quotes: 1
quotes: 1
```

and to allow double quotes for attributes selectors?

```scss
[type="text"] {}
```

Actually, I get this message

```
Strings must use single quotes  quotes
```

Thanks a lot!

-----

What version of Sass Lint are you using? __1.10.2__

Please include any relevant parts of your configuration
```yaml
  quotes: 1
  attribute-quotes: 1
```

What did you do? Please include the actual source code causing the issue.

```scss
[type="text"] {}
```

What did you expect to happen? __no error__

What actually happened? Please include any error messages given to you by Sass Lint.  __Strings must use single quotes__

If you're using a IDE plugin have you tried the CLI too? __Yes, both__

**Comment by thierrymichel** [comment]
As mentioned [here](https://github.com/sasstools/sass-lint/pull/1035) another option would be to use quotes only for values (traverseByType('value'), then check 'string' types) and use attributes-quotes with a new style property…

**Comment by thierrymichel** [comment]
Finally, doesn't make sense to mix single/double quotes in my sass files (as CSS support both).
